types/react-phone-number-input/input.d.ts

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+
+export interface PhoneInputProps extends Omit<React.InputHTMLAttributes<string>, 'onChange'> {
+    /**
+     * If country is specified then the phone number can only be input in "national"
+     * (not "international") format, and will be parsed as a phone number belonging
+     * to the country. Must be a supported country code.
+     * @example "US"
+     */
+    country?: string | undefined;
```

Comment:
I don't think that `| undefined` is needed here and in next props.
`country?: string` means that country could be `undefined`!

Reply:
I think what you're suggesting is that these two lines are functionally the same?
```
country?: string;
country?: string | undefined;
```
While they are at present functionally the same, with the release of [Exact Optional Property Types](https://devblogs.microsoft.com/typescript/announcing-typescript-4-4-beta/#exact-optional-property-types) feature of TS4.4 they're no longer equivalent.

Since the TS maintainers made this change to the main types, I did the same to the input types.

Reply:
Good to know!
But any way I see `country?: string;` as a better choice for React props. 
```js
<PhoneInput
        country={undefined}
    />
```
is better to through error at this, because `country={undefined}` just adds cognitive load.

Reply:
An argument could be made that the types don't force you to add the cognitive load of below, they just enable you to add it yourself.
```
<PhoneInput
        country={undefined}
    />
```

But I'm fine removing the option. I did the same to the change made on [54351](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/54351) by @sandersn so both files are consistent.

I explicitly left `value` as allowed to be undefined, as the package docs include undefined in their examples

Reply:
The change I made was automated for backward compatibility so that TS 4.4 with exactOptionalPropertyTypes would behave the same as TS 4.3. I agree that removing it is the right thing here (and in most react packages as far as I understand).